src/Illuminate/Support/Str.php

```diff
@@ -39,7 +39,7 @@ class Str
      */
     public static function ascii($value)
     {
-        return StaticStringy::toAscii($value);
+        return (string) Stringy::create($value)->toAscii();
```

Comment:
This has to be cast to a string to retain the old behavior of returning a string rather than a Stringy object.

This line of code is an inline copy of 1.8's `StaticStringy::toAscii()` as this seemed least likely to introduce bugs.